src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -166,6 +166,17 @@ DlgPrefDeck::DlgPrefDeck(QWidget * parent, MixxxMainWindow * mixxx,
     connect(checkBoxSeekToCue, SIGNAL(toggled(bool)),
             this, SLOT(slotJumpToCueOnTrackLoadCheckbox(bool)));
 
+    // Double-tap Load to clone a deck via keyboard or controller ([ChannelN],LoadSelectedTrack)
+    // If not present in the config, set the default value
+    if (!m_pConfig->exists(ConfigKey("[Controls]","CloneDeckOnLoadDoubleTap"))) {
```

Comment:
This can be done whenever reading the value.

Reply:
done.